src/natcap/invest/wind_energy.py

```diff
@@ -1129,9 +1129,9 @@ def execute(args):
             if calc_grid_dist_without_land:
                 # Calculate distance raster without land points provided
                 final_dist_task = task_graph.add_task(
-                    func=_calculate_grid_dist_on_raster,
```

Comment:
`_calculate_grid_dist_on_raster` was essentially a copy of `_create_distance_raster` once the latter function used euclidean distances.